app/elm/Combat.elm

```diff
@@ -93,9 +126,14 @@ showAllies allies =
         (List.map Creature.showSprite allies)
 
 
-showEnemy : Creature -> Html Msg -> Creature
-showEnemy enemy =
+showEnemy : Bool -> Creature -> Html Msg
+showEnemy isPlayerTurn enemy =
     div []
         [ Creature.showSprite enemy
-        , div [] [ button [ onClick PlayerAttack enemy ] [ text "attack!" ] ]
+        , div []
+            [ if isPlayerTurn then
+                button [ onClick (PlayerAttack enemy) ] [ text "attack!" ]
```

Comment:
You were missing some parentheses in the original:

`onClick PlayerAttack enemy`

I don't know what it is about `onClick`, but I also often have this thinko when I'm changing from a message with no arguments to one that takes an argument.